docs/release-notes/Language.md

```diff
@@ -0,0 +1,21 @@
+﻿---
+category: Release Notes
+categoryindex: 600
+index: 2
+title: F# Language
+---
+
+# F\# Language
+
+## Unreleased
+
+### Added
+
+* Better generic unmanaged structs handling. ([Language suggestion #692](https://github.com/fsharp/fslang-suggestions/issues/692), [PR #12154](https://github.com/dotnet/fsharp/pull/12154))
+* Bidirectional F#/C# interop for 'unmanaged' constraint. ([PR #12154](https://github.com/dotnet/fsharp/pull/12154))
```

Comment:
How do I map in which language version this starts to be available?

If such "mapping info" is to be added later, which process (who,when,how) makes that a reality?

Reply:
I guess, that would be a follow-up action after a minor release.
Or when the next version is listed in https://github.com/dotnet/fsharp/blob/e185f9bea58612e67f3a65a17501ddaa8ccf9547/src/Compiler/Facilities/LanguageFeatures.fs#L91-L100

This would probably also benefit from a `## Preview` -> `## 8.1 - Frozen` -> `## 8.1 - 2024-02-04` transition.